mycroft/res/text/en-us/date_time_test.json

```diff
@@ -0,0 +1,34 @@
+{
+  "test_nice_year": {
+      "1": {"datetime_param": "2017, 1, 31, 13, 22, 3", "bc": "None", "assertEqual": "two thousand and seventeen"},
```

Comment:
This one is OK, although I'd lean towards "two thousand seventeen"; or "twenty seventeen" to sound natural when speaking a year.